SPECS/fluent-bit/fluent-bit.spec

```diff
@@ -38,7 +39,7 @@ Requires:       %{name} = %{version}
 Development files for %{name}
 
 %prep
-%setup -q
+%autosetup -p1
```

Comment:
+1 for autosetup :)

Does this mean for the previous instances `Patch0` was never applied ?

Reply:
Yes, it was not patched.